app/assets/javascripts/hyrax/admin/collection_type/settings.es6

```diff
@@ -0,0 +1,52 @@
+export default class {
+  constructor(element) {
+    this.element = element
+  }
+
+  setup() {
+    // Watch for changes to "sharable" checkbox
+    let sharableInput = this.element.find("input[type='checkbox'][name$='[sharable]']")
+    $(sharableInput).on('change', () => { this.sharableChanged() })
+    this.sharableChanged()
+  }
+
+  // Based on the "sharable" checked/unchecked, enable/disable adjust share_applies_to_new_works checkbox
+  sharableChanged() {
+    let selected = this.element.find("input[type='checkbox'][name$='[sharable]']:checked")
+    let disabled = this.element.find("input[type='checkbox'][name$='[sharable]']").is(':disabled')
+
+    let share_applies_to_work = this.element.find("input[type='checkbox'][name$='[share_applies_to_new_works]']")
```

Comment:
The convention in javascript is to use camel case. Can we initialize this in `setup()` as `this.shareAppliesToWork`?